meeting.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="https://img.icons8.com/?size=100&id=995Ah2Szp29l&format=png&color=000000" type="icon/x-image">
+    <title>Webex Lyte</title>
+    <link rel="stylesheet" href="./meeting.css" type="text/css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.0/dist/umd/popper.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+
+</head>
+<body id="body">
+        <!-- Bootstrap Modal -->
+<div class="modal fade" id="joinMeetingModal" tabindex="-1" role="dialog" aria-labelledby="joinMeetingModalLabel" aria-hidden="true" data-backdrop="static">
+    <div class="modal-dialog modal-lg" role="document">
+      <div class="modal-content" id="modal1">
+        <div class="modal-body">
+          <!-- Remote Video -->
+          <video id="remoteVideo" autoplay></video>
+          <audio id="remoteAudio" autoplay hidden></audio>
+
+          <!-- Leave and Share Buttons -->
+          <div class="d-flex justify-content-between mt-2">
+            <button id="leaveMeetingButton"><i class="fa-solid fa-xmark"></i></button>
+            <button id="share" class="btn btn-primary mt-2" title="Share"><i class="fa-solid fa-arrow-up-from-bracket"></i></button>
+            
+          </div>
+          <!-- Local Video ,Start/Stop Button ,Unmute/Mute Button -->
+          <div class="local-video-container" draggable="true">
+            <video id="localVideo" autoplay muted></video>
+            <img src="https://img.icons8.com/?size=100&id=10343&format=png&color=000000" id="localvideoimage">

```

Comment:
What is this image? Why do we need it?

Reply:
![Screenshot 2024-06-18 123410](https://github.com/mkesavan13/webex-lyte/assets/46831391/b07a3c9c-a8c9-47f0-a0c1-7f9baf2b6ca5)
Here you can observe the black Video Tag that was the image url instead of using icon

Reply:
I don't think we need an image. Let us change this to font awesome icons

Reply:
By using Font Awesome Icons We cannot Set Size Of Icon

Reply:
Updated